src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -169,7 +170,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 					);
 				}
 
-				$numericEntityId = $dataValue->getEntityId()->getNumericId();
+				$entityId = $dataValue->getEntityId();
+
+				if ( !( $entityId instanceof Int32EntityId ) ) {
+					throw new LogicException(
+						$entityId->getSerialization() . ' expected to be Int32EntityId'
```

Comment:
I would suggest an error message similar to the one above, and talk about a misconfiguration.